azure-communication-ui/calling/publish.gradle

```diff
@@ -91,7 +91,7 @@ publishing {
             version "$azure_calling_sdk_version"
 
             // Add the aar file as an artifact
-            artifact("/Users/inderpalsinghaulakh/Downloads/aar/azure-communication-calling-2.7.0-beta.1.5.aar")
+            artifact("/Users/inderpalsinghaulakh/Downloads/azure-communication-calling-2.7.0-beta.1.13.aar")
 
```

Comment:
When we next touch this, we should use $azure_calling_sdk_version in the name, instead of manually updating.